Omnibus.md

```diff
@@ -3422,4 +3422,304 @@ This spell prevents Undead from attacking the spellcaster while it is active.
+
+Proposals must include the name of the person submitting it and contact
+information for that person. An email address is acceptable for this
+purpose.                        
+
+Those who attend the Players' Meeting shall be permitted to select one
+representative (not an EH) to attend the Event Holders' Council. This
+representative will have the same voting ability at the meeting as an EH, and
+may accept amendments to player proposals submitted at the Players' Meeting.
+
+## 7.2 Becoming A Realms Event-Holder ## {#becoming-a-realms-event-holder}
+
+To attempt to become an Event Holder you must first throw an event with your
+name listed as an Event Holder (see bullets below).
+
+To be a Realms legal event, the following conditions must apply:
+
+* It should be open to the public.
+* It needs to have been advertised with full event information at least two
+  weeks before the date of the event. Advertisement must include at least one
+  of the following means: Publication in The *View from Valehaven* or
+  *Creathorne Chronicles*, posted on the *RealmsNet* Event Calendar, or sent
+  through a mass mailing or electronic mailing, which must include the known
+  addresses of all Realms EHs who have held Realms events in the previous year.
+* The location and the price of the event must be announced at least two weeks
+  before the event is scheduled to take place. Any significant rule changes,
+  especially those affecting normally legal powers and abilities of PCs, must
+  be announced in the advertisement or mailing.
+* The names of the EHs, at most one per day, must be announced at least 5 days
+  before the event is scheduled to take place.
+* A site must be confirmed before an event can be advertised as a legal Realms
+  event.
+* The event must be at least six hours long.
+* At least 30 people (including marshals and NPCs) must attend and the EH must
+  be able to prove attendance through signatures.
+* Players must have the option of playing their normal PCs, within such bounds
+  and restrictions as previously advertised by the EH. Players who do so,
+  despite any such advertised rule changes which may adversely affect their PC,
+  are presumed to freely accept such changes or restrictions.
+* The rules must be based either on the current Omnibus to the Realms or on
+  another rule system approved at the most recent Event Holders’ Council.
+* If the EH(s) are under the age of 18, they must have an adult(s) either as a
+  co-EH(s) or as a Listed Contact(s) in the event description. One of the said
+  adults should be on-site for the event.
+* An EH's PC may not benefit in any way from their event. Their PC may not take
+  possession of any stealable item released at their event for more than one
+  day. They may not receive spell credit as a result of attending their own
+  event.
+* In order for the event to be legal the event must abide by the valid rulings
+  of the Arbitration Committee.
```

Comment:
This should link to the Arbitration Committee section.  The link should be `#arbitration-committee`, but I haven't gotten down that far yet to see if that's what it actually is.

Reply:
Yep, that's the link.